IBPSA/Fluid/Sensors/BaseClasses/PartialAbsoluteSensor.mo

```diff
@@ -29,6 +29,17 @@ IBPSA.Fluid.Sensors.BaseClasses.PartialFlowSensor</a>.
 revisions="<html>
 <ul>
 <li>
+September 7, 2019, by Michael Wetter:<br/>
+Changed
+<code>port(redeclare package Medium=Medium, m_flow(min=0, max=0))</code>
+to
+<code>port(redeclare package Medium=Medium, m_flow(min=0))</code>
+to avoid in Dymola 2019FD01 beta1 the message
+\"port.m_flow has the range [0,0] - which is suspicious since the max-value should be above the min-value\"
```

Comment:
What a lame error! max=min=0 is completely reasonable. I'll report this.

Reply:
I already reported it to the Dymola developers. I think this is a corner case as Modelica does not allow comparing reals for equality due to rounding errors, and max m_flow =min m_flow =0 implies m_flow == 0 which is not allowed as a test.